### PR TITLE
11.6.0b1

### PR DESCRIPTION
**Pull request by xiangyan99**
11.5.0 will not have all features available in this version. Hence re-version it to 11.6.0b1.

### CODE REVIEW

File: sdk/search/azure-search-documents/setup.py [hunk 1]
```diff
@@ -63,7 +62,7 @@
             "azure.search",
         ]
     ),
-    python_requires=">=3.7",
+    python_requires=">=3.8",
```
Comment (pvaneck) [inline sdk/search/azure-search-documents/setup.py:65]: Should also update the README.md and samples/README.md with the minimum Python bump, and mentioning in the Changelog that Python 3.7 is no longer supported (though this should probably be in the `11.5.0` changelog).

**Review by pvaneck — APPROVED** [review]
Do you plan on releasing this beta first then the GA (`11.5.0`)?

**Comment by xiangyan99** [comment]
> Do you plan on releasing this beta first then the GA (`11.5.0`)?

Yes. We plan to GA in March or April.

**Comment by JetterMcTedder** [comment]
@xiangyan99 when might 11.6.0b1 be available to download? Thanks!

**Comment by chris-the-wiz** [comment]
for all those who got here having problem importing `AzureOpenAIParameters` - it is now called `AzureOpenAIVectorizerParameters`

**Pull request merged**